### PR TITLE
lookup: skip eslint on Node.js 10 for now

### PR DESCRIPTION
**Pull request by BridgeAR**
The actual issue got resolved in a eslint 5.0.0-alpha version but it
will take a while until 5.0.0 is officially released as stable. Therefore, let
us just skip eslint for now.

##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [x] documentation is changed or added
- [x] contribution guidelines followed [here](https://github.com/nodejs/citgm/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by richardlau** [comment]
Do you have a link for the issue being referred to?

**Comment by BridgeAR** [comment]
Issue: https://github.com/eslint/eslint/issues/10111
Fix: https://github.com/eslint/eslint/pull/10182

File: lib/lookup.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
   "eslint": {
     "prefix": "v",
     "flaky": ["v7", "s390", "ubuntu"],
-    "skip": ["v8", "ppc"],
+    "skip": ["v8", "ppc", ">9"],
```
Comment (MylesBorins) [inline lib/lookup.json]: could this be `[">v8", "ppc"],?

Comment (BridgeAR) [reply lib/lookup.json]: I guess it should be `[">8", "ppc"]`? I am not sure what labels all exist but I doubt that the `v` is necessary.

**Comment by BridgeAR** [comment]
Comment addressed.

File: lib/lookup.json [hunk 2]
```diff
@@ -105,8 +105,8 @@
   },
   "eslint": {
     "prefix": "v",
-    "flaky": ["v7", "s390", "ubuntu"],
-    "skip": ["v8", "ppc"],
+    "flaky": ["s390", "ubuntu"],
+    "skip": ["ppc", ">8"],
```
Comment (targos) [inline lib/lookup.json]: is it also broken on Node 9?

**Comment by MylesBorins** [comment]
landed in 9b39eb1607

In future feel free to land changes to the lookup without review as mentioned in the [collaborator guide](https://github.com/nodejs/citgm/blob/master/COLLABORATORS.md). Just make sure to open a PR for documentation purposes

**Comment by BridgeAR** [comment]
@MylesBorins I have no commit rights yet.

**Comment by MylesBorins** [comment]
This is an over site https://github.com/nodejs/Release/issues/342

**Pull request closed without merging**